src/chrome/content/rules/Elster.de.xml

```diff
@@ -0,0 +1,16 @@
+<ruleset name="Elster.de">
+	<target host="www.elster.de" />
+	<target host=    "elster.de" />
+	<target host="www.elsteronline.de" />
+	<target host=    "elsteronline.de" />
+
+	<test url="https://elster.de/" />
+	<test url="https://elsteronline.de/" />
+
+	<rule from="^https?://elster\.de/"
+		to="https://www.elster.de/" />
+	<rule from="^https?://elsteronline\.de/"
+		to="https://www.elsteronline.de/" />
```

Comment:
http://elster.de/ needs rule in line 10 to add www.
https://elster.de/ needs rule in line 10 to add www.
http://www.elster.de/ needs rule in line 14.
http://elsteronline.de/ needs rule in line 12 to add www.
https://elsteronline.de/ needs rule in line 12 to add www.
http://www.elsteronline.de/ needs rule in line 14.

Please explain which rule you think is not needed and why.